velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -236,14 +236,41 @@ std::vector<std::string> AggregationFuzzerBase::generateKeys(
 std::vector<std::string> AggregationFuzzerBase::generateSortingKeys(
     const std::string& prefix,
     std::vector<std::string>& names,
-    std::vector<TypePtr>& types) {
+    std::vector<TypePtr>& types,
+    const bool hasRowNumberKey,
```

Comment:
Does hasRowNumberKey need to be passed as a parameter ? Can we find it from the names vector instead ?